### DISCUSSION TITLE
Release v1.0.19 / 5.55.19

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1815] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build is a maintenance release, fixing various issues

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# ChangeLog

### Added
- added drag and drop support for groups [#1775](https://github.com/sandboxie-plus/Sandboxie/issues/1775)
- added del key support to the box view for all entry types [#1779](https://github.com/sandboxie-plus/Sandboxie/issues/1779)
- added warning when trying to run explorer.exe in a box with OpenCOM [#1716](https://github.com/sandboxie-plus/Sandboxie/issues/1716)

### Fixed
- fixed crash issue in the sandman ui [#1772](https://github.com/sandboxie-plus/Sandboxie/issues/1772)
- fixed issue some installers when EnableObjectFiltering is enabled [#1795](https://github.com/sandboxie-plus/Sandboxie/issues/1795)
- fixed to allow NtCreateSymbolicLinkObject to be used safely in the sandbox
- added workaround for a vivaldi hooking issue [1783](https://github.com/sandboxie-plus/Sandboxie/issues/1783)
-- Note: its a very provisional fix hence it can be disabled with UseVivaldiWorkaround=n
- fixed registry issue with snapshots [#1782](https://github.com/sandboxie-plus/Sandboxie/issues/1782)
- fixed issue with box grouping [#1778](https://github.com/sandboxie-plus/Sandboxie/issues/1778) [#1777](https://github.com/sandboxie-plus/Sandboxie/issues/1777) [#1776](https://github.com/sandboxie-plus/Sandboxie/issues/1776) 
- fixed more issue with box grouping [#1698](https://github.com/sandboxie-plus/Sandboxie/issues/1698) [#1697](https://github.com/sandboxie-plus/Sandboxie/issues/1697)
- fixed issues with snadshot ui [#1696](https://github.com/sandboxie-plus/Sandboxie/issues/1696) [#1695](https://github.com/sandboxie-plus/Sandboxie/issues/1695)
- fixed issue with recovery dialog focus [#1374](https://github.com/sandboxie-plus/Sandboxie/issues/1374)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/1.0.19'>Release v1.0.19 / 5.55.19</a>.</em>

**Comment by BilleBarrett** [comment]
I received this error when I installed Plus on a client's computer I had to switch to Classic so he could run
What needs to be set to fix this error, or what do I have to do to his system
Thanks
Bill
Local State is eligible for quick recovery from Defaultbox
The file was written by: Chrome.exe (8836)
C:\User\XXXX\AppData\local\Google\Chrome\User Data
Browser
D:\Downloads\Juke
C:\User\XXXX\Documents\new folder
C:\User\XXXX\Documents
D:\downloads

**Comment by BilleBarrett** [comment]
It seems that Microsoft Edge Webviewer Runtime was updated yesterday 4/24/2022 100.0.1185.50
This morning when I go to open edge in the Sandbox I just get a white screen, open outside of Sandboxie-Plus everything is OK
I tried to open Edge on another system in the office and it did the same thing
I'm not sure what is wrong here but Chrome seems unaffected
Any suggestions 
Bill 
PS do you need any info
Very sorry, I was stuck in a Microsoft Beta update that was not working, fix it

**Reply by bastik-1001** [reply]
If your problems persist, you may want to create issues for them, so they can be kept track of.
